spring-cloud-dataflow-server-local/src/main/resources/application.yml

```diff
@@ -43,6 +38,18 @@ logging:
 #    resource:
 #      user-info-uri: http://127.0.0.1:9999/me
 
+# UAA Security
+
+security:
```

Comment:
Maybe we could just document the appropriate values for the OAuth2 properties and delete the following entries (with random default values) from local server application config.